uxarray/plot/dataarray_plot.py

```diff
@@ -61,3 +71,168 @@ def datashade(uxda: UxDataArray,
         _cmap = cmap
 
     return tf.shade(aggregated, cmap=_cmap, **kwargs)
+
+
+def _point_raster(uxda: UxDataArray,
+                  backend: Optional[str] = "bokeh",
+                  pixel_ratio: Optional[float] = 1.0,
+                  dynamic: Optional[bool] = False,
+                  precompute: Optional[bool] = True,
+                  projection: Optional[ccrs] = None,
+                  width: Optional[int] = 1000,
+                  height: Optional[int] = 500,
+                  colorbar: Optional[bool] = True,
+                  cmap: Optional[str] = "Blues",
+                  aggregator: Optional[str] = "mean",
+                  interpolation: Optional[str] = "linear",
+                  npartitions: Optional[int] = 1,
+                  cache: Optional[bool] = True,
+                  **kwargs):
+    """Implementation of Point Rasterization."""
+
+    if uxda._face_centered():
+        # data mapped to face centroid coordinates
+        lon = uxda.uxgrid.Mesh2_face_x.values
+        lat = uxda.uxgrid.Mesh2_face_y.values
+    elif uxda._node_centered():
+        # data mapped to face corner coordinates
+        lon = uxda.uxgrid.Mesh2_node_x.values
+        lat = uxda.uxgrid.Mesh2_node_y.values
+    else:
+        raise ValueError(
+            f"The Dimension of Data Variable {uxda.name} is not Node or Face centered."
+        )
+
+    # determine whether we need to recompute points, typically when a new projection is selected
+    recompute = True
+    if uxda._face_centered() == "center":
+        if uxda.uxgrid._centroid_points_df_proj[
+                0] is not None and uxda.uxgrid._centroid_points_df_proj[
+                    1] == projection:
+            recompute = False
+            points_df = uxda.uxgrid._centroid_points_df_proj[0]
+
+    else:
+        if uxda.uxgrid._corner_points_df_proj[
+                0] is not None and uxda.uxgrid._corner_points_df_proj[
+                    1] == projection:
+            recompute = False
+            points_df = uxda.uxgrid._corner_points_df_proj[0]
+
+    if recompute:
+        # need to recompute points and/or projection
+        if projection is not None:
+            lon, lat, _ = projection.transform_points(ccrs.PlateCarree(), lon,
```

Comment:
ccrs is one of the options for projection?
`projection: Optional[ccrs] = None,` 

what other options? hard-coded?

Reply:
The typing here is only for hints, so it doesn't restrict us, but it's expected to be a Cartopy Projection.